File: app/src/main/java/com/holokenmod/GridBorderType.java
package com.holokenmod;

public enum GridBorderType {
    BORDER_NONE,
    BORDER_SOLID,
    BORDER_WARN,
    BORDER_CAGE_SELECTED;

    public boolean isHighlighted() {
        return this == BORDER_WARN || this == BORDER_CAGE_SELECTED;
    }
}

File: gauguin-core/src/main/kotlin/org/piepmeyer/gauguin/grid/GridBorderType.kt
package org.piepmeyer.gauguin.grid

enum class GridBorderType {
    BORDER_NONE,
    BORDER_SOLID,
    BORDER_WARN,
    BORDER_CAGE_SELECTED,
    ;

    val isHighlighted: Boolean
        get() = this == BORDER_WARN || this == BORDER_CAGE_SELECTED
}
